rocAL/source/meta_data/text_file_meta_data_reader.cpp

```diff
@@ -67,17 +67,24 @@ void TextFileMetaDataReader::lookup(const std::vector<std::string> &image_names)
 }
 
 void TextFileMetaDataReader::read_all(const std::string &path) {
-    std::ifstream text_file(path.c_str());
+    std::ifstream text_file(_file_list_path.c_str());
     if (text_file.good()) {
         //_text_file.open(path.c_str(), std::ifstream::in);
         std::string line;
         while (std::getline(text_file, line)) {
-            std::istringstream line_ss(line);
-            int label;
-            std::string image_name;
-            if (!(line_ss >> image_name >> label))
-                continue;
-            add(image_name, label);
+            std::istringstream read_stream(line);
+            std::string file_name;
+            uint file_label;
+            if (!(read_stream >> file_name >> file_label)) {
+                break;
```

Comment:
Do we want to break / continue?

Please retain the previous line changes itself, let us not change

The name of image_name alone you can change to file_name, do not change what was there previously

Reply:
Did you check this?